lib/camping/server.rb

```diff
@@ -149,8 +99,7 @@ def opt_parser
 
     def default_options
       super.merge({
-        :Port => 3301,
-        :database => Options::DB
+        :Port => 3301
```

Comment:
Should this be lowercased? I'm confused as to why it's uppercase.

Reply:
It's uppercased because of convention. I've changed this to lowercase now.

Reply:
Wait. Actually I remember that there is a good reason for this.

`Camping::Server` inherits from `Rackup::Server` and in *Rackup* these keys are capitalized. I suspect it's a legacy decisions that hasn't changed for a really long time.

Reply:
@karloscarweber Got it, thank you for looking into this! I've not seen it capitalized before but since we need to, it's good to have the reason documented.